Buddies/Profile/ProfileVC.swift

```diff
@@ -91,9 +91,8 @@ class ProfileVC: UIViewController, UICollectionViewDelegateFlowLayout {
     
     // Gets topics from the root topic store
     func getTopics(from topicIds: [String]) -> [Topic] {
-        let ids = Array(Set(topicIds))
         let appDelegate = UIApplication.shared.delegate as! AppDelegate
-        let topics = appDelegate.topicCollection.topics
-        return ids.map { id in topics.first { $0.id == id } }.compactMap { $0 }
+        let topics = appDelegate.topicCollection.topics.filter { topicIds.contains($0.id) }
```

Comment:
There was still the bug where you had duplicate topics in your user data - if you add line 94 back in you can also avoid that for existing users, though i expect this PR will fix the root of that issue

Reply:
Since we `filter` with `contains`, only if `topicCollections` has duplicates will we have duplicates.  So this shouldn't be a problem unless our root problem comes back.